### PR TITLE
fix(pci): add error message for region selector in storage add

### PR DESCRIPTION
**Pull request by frenautvh**
See : DTRSD-6467

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html [hunk 1]
```diff
@@ -14,7 +14,12 @@
         data-on-submit="$ctrl.onRegionChange()"
         data-editable="!$ctrl.isLoading"
         name="storage_container_add_region">
+        <oui-message type="error"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html]: I want to say `data-` but I'm not sure anymore 😄

Comment (frenautvh) [reply packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html]: added for concistency

**Comment by antleblanc** [comment]
Hi @frenautvh!

We recently merged #1336 which apply Prettier rules so could you please rebase
your branch on `develop`?

To easily resolve conflicts you can **ignore** `HEAD` changes and only keep yours.

After that you can run:

```sh
$ yarn run lint
```

This command will lint the entire codebase.

or:

Amend your commit to trigger the `pre-commit` hook (prettier will be launched
under the hood).

Thanks!

**Comment by antleblanc** [comment]
@frenautvh Internal ticket has been closed.
Could you please update the status of your Pull Request (if needed)
and also address the conflict?

Thanks!

**Pull request closed without merging**